lib/msgpack.rb

```diff
@@ -1,6 +1,13 @@
 require "msgpack/version"
-begin
-  require "msgpack/#{RUBY_VERSION[/\d+.\d+/]}/msgpack"
-rescue LoadError
-  require "msgpack/msgpack"
+
+if defined?(RUBY_ENGINE) && RUBY_ENGINE == 'jruby'
+  require 'jruby'
```

Comment:
`require "jruby"` also works well and works as same with `require "java"`.
But it seems to be `require "java"` for consistency. I'll fix it.